olive/common/pydantic_v1.py

```diff
@@ -0,0 +1,19 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+"""Pydantic v1 compatibility module.
+
+Pydantic v2 has breaking changes that are not compatible with the current version of Olive.
+Migration Guide: https://docs.pydantic.dev/latest/migration/.
+
+In order to support both versions of Pydantic, we use this module to access pydantic's v1 API.
+"""
+
+try:
+    # pydantic v2
+    from pydantic.v1 import *  # noqa: F403
```

Comment:
would the namespace in v2 is full compatible with exist v1 stuff? did you try it? 

I guess there might be some features are not supported in v2 back-compatibility?